src/sentry/api/endpoints/sentry_app_installation_details.py

```diff
@@ -19,3 +20,22 @@ def delete(self, request, installation):
             request=request,
         )
         return Response(status=204)
+
+    def put(self, request, installation):
+        serializer = SentryAppInstallationSerializer(
+            installation,
+            data=request.data,
+            partial=True,
+        )
+
+        if serializer.is_valid():
+            result = serializer.validated_data
+
+            updated_installation = Updater.run(
```

Comment:
Nitpick (not necessary to change): this can just be `installation`. `Updater` I think is a good enough indicator of what's happening.